queries/elixir/highlights.scm

```diff
@@ -180,39 +180,38 @@
       ] operator: "/" right: (integer) @operator)
   ])
 
-; Module attributes
-(unary_operator 
-  operator: "@" @constant 
-  operand: [
-    (identifier) @constant
-    (call target: (identifier) @constant)])
-
 ; Sigils
-(sigil 
+(sigil
   "~" @string.special
   ((sigil_name) @string.special) @_sigil_name
-  quoted_start: _ @string.special 
+  quoted_start: _ @string.special
   quoted_end: _ @string.special
   ((sigil_modifiers) @string.special)?
   (#not-any-of? @_sigil_name "s" "S"))
 
-(sigil 
+(sigil
   "~" @string
   ((sigil_name) @string) @_sigil_name
-  quoted_start: _ @string 
+  quoted_start: _ @string
   (quoted_content) @string
   quoted_end: _ @string
   ((sigil_modifiers) @string)?
   (#any-of? @_sigil_name "s" "S"))
 
+; Module attributes
+(unary_operator
+  operator: "@"
+  operand: [
+    (identifier)
+    (call target: (identifier) @constant)]) @constant
+
 ; Documentation
-(unary_operator 
-  operator: "@" @comment 
-  operand: (call 
-    target: (((identifier) @comment) @_identifier)
+(unary_operator
+  operator: "@"
+  operand: (call
+    target: ((identifier) @_identifier (#match? @_identifier "doc$")) @comment
```

Comment:
Would a more specific match also work?

```suggestion
    target: ((identifier) @_identifier (#match? @_identifier "^(moduledoc|typedoc|doc)$")) @comment
```

Reply:
if you go for that, using `#any-of?` is better.

Reply:
Fixed